services/localfile/server/localfile.go

```diff
@@ -323,6 +325,31 @@ func setupOutput(a *pb.FileAttributes) (*os.File, *immutableState, error) {
 	return f, immutable, err
 }
 
+// create a directory, set ownership and mode
+func setupDir(a *pb.FileAttributes) (string, *immutableState, error) {
```

Comment:
maybe `createDir()` or `mkDir()` would be a better name?

Reply:
I will go with `createDir()`